lib/action_subscriber.rb

```diff
@@ -80,6 +80,10 @@ def self.start_subscribers
     print_subscriptions
   end
 
+  def self.stop_subscribers!
+    @route_set.cancel_consumers! if @route_set
```

Comment:
Why is there are guard here for `@route_set`, but not other places in this code?